epifor/data/batch.py

```diff
@@ -267,21 +269,32 @@ def export_region_traces(self, er: ExportRegion, out_dir: Path):
         }
         er.data["mitigation_stats"] = gs
 
-    def export_region_estimates(self, er: ExportRegion):
-        # Stats
-        ests = {
-            k: self.region_data.get(er.region.key, {}).get(k)
-            for k in [
-                "JH_Deaths",
-                "JH_Confirmed",
-                "JH_Recovered",
-                "JH_Infected",
-                "FT_Infected",
-            ]
+    def export_region_estimates(self, er: ExportRegion, df):
+
+        columns_list = sorted(set([date.split('_')[1] for date in df.columns]))
+        days = {}
+
+        try:
+            row = df.loc[er.region.key]
+        except Exception as ex:
+            logging.warning(f"Region not in data : {str(ex)}")
```

Comment:
I propose to just catch `KeyError`

```suggestion
        except KeyError as ex:
            logging.warning(f"Region not in data : {str(ex)}")
```

and also there's `logging.exception`, but that may be overkill here

Reply:
ok